cmd/device/device.go

```diff
@@ -1,39 +0,0 @@
-// Copyright © 2019 VMware, INC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package device
-
-import (
-
-	"github.com/edgexfoundry/edgex-cli/cmd/device/add"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/list"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/rm"
-	"github.com/edgexfoundry/edgex-cli/cmd/device/update"
-	
-	"github.com/spf13/cobra"
-)
-
-// NewCommand returns the device command of type cobra.Command
-func NewCommand() *cobra.Command {
```

Comment:
This file `cmd/device/device.go` should be present that is why 
make test fails with 

> go: finding module for package github.com/edgexfoundry/edgex-cli/cmd/device
> cmd/root.go:25:2: no matching versions for query "latest"
> Makefile:29: recipe for target 'test' failed
> make: *** [test] Error 1

Reply:
Rebase could help with this issue and will make the PR contains just the files you have really changed

Reply:
make test still fails with me ..but I doubt that the problem is in my local environment. I will mege the PR and later investigate why  make test do not succeed with me.